CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Improvements:
  * `Address.isContract`: switched from `extcodesize` to `extcodehash` for less gas usage. ([#1802](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1802))
+ * `ERC721`:  ERC721 test suite was missing getApproved function test. Added that.([#1148](https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1148))
```

Comment:
I don't think we're documenting test suite changes in the changelog. Can you remove this please?

I realize now that the pull request template on GitHub asks that you always add a changelog entry, sorry about that, we'll have to change it.

Reply:
Just out of curiousity, is there a reason for that? Shouldn't we be aware what went in with every commit?